src/app/cart/cart.tpl.html

```diff
@@ -32,7 +32,7 @@ <h3 class="panel-name" translate>Your Gift Cart</h3>
                         <span class="giftsum-frequency giftsum-title">{{i.frequency}}</span>
                         <span class="giftsum-date giftsum-detail" ng-if="i.frequency !== 'Single'">
                           <translate>Starts on:</translate>
-                          {{$ctrl.donationStartDate(i.config['start-month'], i.config['start-day']) | date:'MMM d'}}
+                          {{i.giftStartDate | date:'MMM d, yyyy'}}
```

Comment:
This seems to be missing a space. The view looks like `Starts on:Oct 4, 2016`. If you move the translate up to be an attribute on the span that might work.